grammar.js

```diff
@@ -723,6 +724,59 @@ module.exports = grammar({
       ),
     ),
 
+    comment_text: $ => $._literal_string,
+
+    comment_statement: $ => seq(
+      $.keyword_comment,
+      $.keyword_on,
+      $._comment_target,
+      $.keyword_is,
+      choice(
+        $.keyword_null,
+        $.comment_text,
+      ),
+    ),
+
+    _comment_target: $ => choice(
+      // TODO: access method
+      // TODO: aggregate
+      $.cast,
+      // TODO: collation
+      seq($.keyword_column, $.object_reference),
+      // TODO: constraint (on domain)
+      // TODO: conversion
+      seq($.keyword_database, $.identifier),
+      // TODO: domain
+      // TODO: extension
+      // TODO: event trigger
+      // TODO: foreign data wrapper
+      // TODO: foreign table
+      // TODO: function
+      seq($.keyword_index, $.object_reference),
+      // TODO: large object
+      seq($.keyword_materialized, $.keyword_view, $.object_reference),
+      // TODO: operator (|class|family)
+      // TODO: policy
+      // TODO: (procedural) language
+      // TODO: procedure
+      // TODO: publication
+      seq($.keyword_role, $.object_reference),
+      // TODO: routine
+      // TODO: rule
+      seq($.keyword_schema, $.object_reference),
```

Comment:
```suggestion
      seq($.keyword_schema, $.identifier),
```
schema can only be the one name